framework/src/ble_manager/ble_manager_api.c

```diff
@@ -343,6 +343,15 @@ ble_result_e ble_server_charact_indicate(ble_attr_handle attr_handle, ble_conn_h
 	RETURN_RESULT(res, msg);
 }
 
+ble_result_e ble_server_get_indicate_pending_count(ble_conn_handle con_handle, uint8_t *count)
+{
```

Comment:
Pls perform NULL check for `count` before using it.

Reply:
Added a NULL for count